src/sentry/tasks/backfill_seer_grouping_records.py

```diff
@@ -0,0 +1,333 @@
+                    "project_id": project.id,
+                    "group_id": group_id,
+                    "event_id": event_id,
+                }
+                logger.info("tasks.backfill_seer_grouping_records.event_lookup_error", extra=extra)
+                continue
+            except KeyError:
+                extra = {
+                    "organization_id": project.organization.id,
+                    "project_id": project.id,
+                    "group_id": group_id,
+                }
+                logger.info("tasks.backfill_seer_grouping_records.no_group_hash", extra=extra)
+                continue
+
+    return bulk_group_data_stacktraces
+
+
+@metrics.wraps(f"{BACKFILL_NAME}.lookup_event_bulk", sample_rate=1.0)
+def lookup_group_data_stacktrace_bulk(
+    project: Project, rows: list[GroupEventRow], messages: dict[int, str], hashes: dict[int, str]
+) -> tuple[set[str], set[str], GroupStacktraceData]:
+    project_id = project.id
+    node_id_to_group_data = {
+        Event.generate_node_id(project_id, event_id=row["event_id"]): (
+            row["event_id"],
+            row["group_id"],
+        )
+        for row in rows
+    }
+
+    try:
+        bulk_data = nodestore.backend.get_multi(list(node_id_to_group_data.keys()))
+    except (ServiceUnavailable, DeadlineExceeded):
+        time.sleep(2)
+        try:
+            bulk_data = nodestore.backend.get_multi(list(node_id_to_group_data.keys()))
+        except (ServiceUnavailable, DeadlineExceeded):
+            time.sleep(4)
+            try:
+                bulk_data = nodestore.backend.get_multi(list(node_id_to_group_data.keys()))
+            except (ServiceUnavailable, DeadlineExceeded) as e:
+                bulk_data = {}
+                extra = {
+                    "organization_id": project.organization.id,
+                    "project_id": project.id,
+                    "group_data": json.dumps(rows),
+                    "error": e.message,
+                }
+                logger.info(
+                    "tasks.backfill_seer_grouping_records.bulk_event_lookup_exception",
+                    extra=extra,
+                )
+
+    group_data = []
+    stacktrace_strings = []
+    bulk_event_ids = set()
+    invalid_event_ids = set()
+    with sentry_sdk.start_transaction(op="embeddings_grouping.get_latest_event"):
+        for node_id, data in bulk_data.items():
+            if node_id in node_id_to_group_data:
+                event_id, group_id = (
+                    node_id_to_group_data[node_id][0],
+                    node_id_to_group_data[node_id][1],
+                )
+                event = Event(event_id=event_id, project_id=project_id, group_id=group_id)
+                event.data = data
+                if event and event.data and event.data.get("exception") and hashes.get(group_id):
+                    grouping_info = get_grouping_info(
+                        None, project=project, event_id=event.event_id, event=event
```

Comment:
As of https://github.com/getsentry/sentry/pull/69932, we only have to pass in the event.

```suggestion
                        None, project=project, event=event
```